extensions/cornerstone/src/commandsModule.ts

```diff
@@ -2063,27 +2064,30 @@ function commandsModule({
         });
       }
     },
-    increaseBrushSize: () => {
+    _handleBrushSizeAction: (action: 'increase' | 'decrease') => {
```

Comment:
I think this should be moved to outside the `actions` object since it itself is not an action. Notice there are other utility functions above the `actions` object.